manifests/site.pp

```diff
@@ -22,6 +22,7 @@
     include profile::ssh::hostbased_auth::client
     include profile::ceph::client
     include profile::ccsudo::sudoers
+    include profile::localscratch::ephemeraldisk
```

Comment:
`File['/localscratch']` is created by `profile::slurm::node`, so it is missing on login node.

Unless there is a need for /localscratch on login-node that I am not aware of, remove this line.